### ISSUE TITLE
[BUG]: YT Error (Sign in to confirm you’re not a bot)

### ISSUE THREAD

**Issue by ZuherLaith** [PerformanC/NodeLink#78]
### NodeLink version

2.2.0 -- latest

### Description

Basically YT source isn't behavioring as expected and is not functioning.

Note: If authorization is required to put this issue to an end, it should have require easy info to do so.

This issue has been going for the past weeks.

### Client

FL 2.4.1 -- latest

### Steps to reproduce

Request any YT link and monitor your logs to see what it outputs.

### Confirmations

- [X] My environment meets the minimum requirements.
- [X] I have verified that this is not a duplicate issue.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by UnschooledGamer** [comment]
@ZuherLaith We're Already aware of it, And I think 🤔 Pedroo already has ways for Authorization. And Another fix/way was partially added. Those probably fix the problem. I don't Know because I haven't occurred these Issues.

@ThePedroo

**Comment by ZuherLaith** [comment]
> Pedroo already has ways for Authorization. And Another fix/way was partially added. Those probably fix the problem. I don't Know because I haven't occurred these Issues.
> 
Sadly Pedro's authorization method requires a rooted android device, I already tried it on nox android emulator, didn't work.
Also, this is a non-friendly user fix, this problem can appear to any other user making it challenging to fix the issue.
Therefore, another solution or easier one is required if possible.

**Comment by UnschooledGamer** [comment]
> > Pedroo already has ways for Authorization. And Another fix/way was partially added. Those probably fix the problem. I don't Know because I haven't occurred these Issues.
> 
> Sadly Pedro's authorization method requires a rooted android device, I already tried it on nox android emulator, didn't work. Also, this is a non-friendly user fix, this problem can appear to any other user making it challenging to fix the issue. Therefore, another solution or easier one is required if possible.

Cannot do anything except waiting for An Fix.

**Comment by ThePedroo** [comment]
> > Pedroo already has ways for Authorization. And Another fix/way was partially added. Those probably fix the problem. I don't Know because I haven't occurred these Issues.
> > 
> Sadly Pedro's authorization method requires a rooted android device, I already tried it on nox android emulator, didn't work.
> Also, this is a non-friendly user fix, this problem can appear to any other user making it challenging to fix the issue.
> Therefore, another solution or easier one is required if possible.
> 

That's wrong, that method doesn't require a rooted device. Well, one of the approaches is through a rooted device, but you can also use an Android emulator, and use HTTPToolkit.

**Comment by 1Lucas1apk** [comment]
Fixed in rewrite adding oauth2 + TV clients

https://github.com/PerformanC/NodeLink/blob/rewrite/src%2Fsources%2Fyoutube%2FOAuth.js
